src/systems/filecoin_vm/sysactors/init_actor.go

```diff
@@ -2,92 +2,134 @@ package sysactors
 
 import addr "github.com/filecoin-project/specs/systems/filecoin_vm/actor/address"
 import actor "github.com/filecoin-project/specs/systems/filecoin_vm/actor"
-
-// import vmr "github.com/filecoin-project/specs/systems/filecoin_vm/runtime"
+import vmr "github.com/filecoin-project/specs/systems/filecoin_vm/runtime"
 import exitcode "github.com/filecoin-project/specs/systems/filecoin_vm/runtime/exitcode"
-
-// import st "github.com/filecoin-project/specs/systems/filecoin_vm/state_tree"
-// import util "github.com/filecoin-project/specs/util"
-
-func (a *InitActorCode_I) Constructor(rt Runtime) {
+import util "github.com/filecoin-project/specs/util"
+import msg "github.com/filecoin-project/specs/systems/filecoin_vm/message"
+import ipld "github.com/filecoin-project/specs/libraries/ipld"
+
+////////////////////////////////////////////////////////////////////////////////
+// Boilerplate
+////////////////////////////////////////////////////////////////////////////////
+type InvocOutput = msg.InvocOutput
+type Runtime = vmr.Runtime
+type Bytes = util.Bytes
+
+func (a *InitActorCode_I) State(rt Runtime) (vmr.ActorStateHandle, InitActorState) {
+	h := rt.AcquireState()
+	stateCID := h.Take()
+	stateBytes := rt.IpldGet(ipld.CID(stateCID))
+	if stateBytes.Which() != vmr.Runtime_IpldGet_FunRet_Case_Bytes {
+		rt.Abort("IPLD lookup error")
+	}
+	state := DeserializeState(stateBytes.As_Bytes())
+	return h, state
+}
+func Release(rt Runtime, h vmr.ActorStateHandle, st InitActorState) {
+	checkCID := actor.ActorSubstateCID(rt.IpldPut(st.Impl()))
+	h.Release(checkCID)
+}
+func UpdateRelease(rt Runtime, h vmr.ActorStateHandle, st InitActorState) {
+	newCID := actor.ActorSubstateCID(rt.IpldPut(st.Impl()))
+	h.UpdateRelease(newCID)
+}
```

Comment:
@jzimmerman this boilerplate is tedious/error prone in every actor. (though 6 is probably fine :) )